fov_processing_pipeline/stats/z_intensity_profile.py

```diff
@@ -111,4 +115,29 @@ def plot(
     plt.savefig(save_path)
     plt.close()
 
+    # make plot of means for each channel
+    plt.figure()
+    for color, ch in zip(colors, range(len(columns))):
+        df_ch = df_pm[df_pm['ch'].str.contains(str(ch))]
+        z_vals = np.arange(min(df_ch['z']), max(df_ch['z'])+1)
+
+        # get mean and standard deviation by z index
+        means = []
+        stds = []
+        for z in z_vals:
+            means.append(np.mean(df_ch[df_ch['z']==z]['intensity']))
+            stds.append(np.std(df_ch[df_ch['z']==z]['intensity']))
+
+        # plot mean as a solid line and shade area +- standard deviation relative to mean
+        plt.plot(z_vals, means, color=color, label=df_ch['ch'].iloc[0])
+        y1 = [means[i]-stds[i] for i in range(len(means))]
+        y2 = [means[i]+stds[i] for i in range(len(means))]
+        plt.fill_between(z_vals, y1, y2, color=color, alpha=0.1)
+    plt.legend()
+    plt.xlabel("z-position{}".format(x_label_suffix))
+    plt.ylabel("intensity{}".format(y_label_suffix))
+    ext_ind = save_path.rfind(".")
+    plt.savefig(save_path[:ext_ind]+"_mean"+save_path[ext_ind:])
```

Comment:
The parsing of the save_path string is sort of odd. 

I'd suggest either adding a flag to do the "mean" plot instead of the per-fov plot, passing in an additional path for the other plot, or making a new plot function entirely (also in this module) that could be called.